### ISSUE TITLE
Why not only use  'stage_5_out' to  calculate l2_loss as the total loss?

### ISSUE THREAD

**Issue by xiaqing10** [edvardHua/PoseEstimationForMobile#111]
In your train.py:
`    for idx, pred_heat in enumerate(pred_heatmaps_all):
        loss_l2 = tf.nn.l2_loss(tf.concat(pred_heat, axis=0) - input_heat, name='loss_heatmap_stage%d' % idx)
        losses.append(loss_l2)
`
use stage_(0-5)_out to calculate loss, but the real output is stage_5_out ,so ,i guess could use only stage_5_out as the total loss , maybe i was wrong , could you explain?
thanks.

**Comment by xiaqing10** [comment]
intermediate supervision.
Ok , i know.
